Doc/c-api/unicode.rst

```diff
@@ -971,6 +971,12 @@ These are the UTF-8 codec APIs:
    returned buffer always has an extra null byte appended (not included in
    *size*), regardless of whether there are any other null code points.
 
+   If *size* is NULL and the *unicode* string contains embedded null
```

Comment:
The wording differs from the one for `PyUnicode_AsWideCharString()`. It would be better to have the same wording for the same behavior, so the user do not need to search non-existing differences.